pkg/controllers/sidecar/identity/member.go

```diff
@@ -63,27 +66,43 @@ func Retrieve(ctx context.Context, name, namespace string, kubeclient kubernetes
 	}, nil
 }
 
-func (m *Member) GetSeeds(ctx context.Context, kubeClient kubernetes.Interface) ([]string, error) {
-	var services *corev1.ServiceList
-	var err error
+// GetSeed returns a single node contact point used for bootstrapping.
+// A single node is enough because bootstrap process will fail if any of the nodes in cluster is down.
+func (m *Member) GetSeed(ctx context.Context, coreClient v1.CoreV1Interface) (string, error) {
+	pods, err := coreClient.Pods(m.Namespace).List(ctx, metav1.ListOptions{
+		LabelSelector: labels.SelectorFromSet(labels.Set{
+			naming.ClusterNameLabel: m.Cluster,
+		}).String(),
+	})
+	if err != nil {
+		return "", err
+	}
 
-	sel := fmt.Sprintf("%s,%s=%s", naming.SeedLabel, naming.ClusterNameLabel, m.Cluster)
+	// Return static IP of itself to bootstrap first node.
+	if len(pods.Items) == 1 && pods.Items[0].Name == m.Name {
```

Comment:
if a pod is evicted in the meantime, it won't be listed here - that doesn't meant it's alone. I suppose the desired member count from ScyllaCluster would be more fitting